### PR TITLE
Manually add metadata to submission when not using a DOI

### PR DESCRIPTION
**Pull request by jabrah**
#655 

* User can now enter author info in the metadata step
* Manually entered article title and journal data is added to the metadata blob
* Manually entered article title and journal info is prepopulated into the metadata step

Addition:
* `issn-map` should not appear in either the submission review _or_ submission details pages

## Testing
* Load up docker and login as normal
* Start a new submission
* ### **Do not enter a DOI** 
* Enter stuff for _article title_ and select a journal. Click Next
* Progress until you get to the Details step. 
  * Article title, Journal ISSN should appear correctly
  * User should be able to add authors here (hint: add a user)
* Progress with the rest of the submission
  * **Make sure you have your developer tools open for you browser, opened to the Network tab**. This way you can monitor the outbound requests
* Click the final Submit button
  * In the dev tools, you will see a `submissions` request. The response of this will have the new submission ID in Fedora. You can directly navigate to this URL to see the submission object in Fedora
* Bring up the specific submission in Fedora and inspect it's `metadata` property - this will be a stringified JSON blob. 
  * Article title, journal name, issn, and issn-map should all be present
  * If you had added an author, that author should also be present

### CODE REVIEW

File: app/components/workflow-basics.js [hunk 1]
```diff
@@ -197,7 +197,20 @@ export default Component.extend({
...
+      let doiInfo = this.get('doiInfo');
+      doiInfo = {
+        'journal-title': journal.get('journalName'),
+        ISSN: journal.get('issns')
+      };
+
+      const nlmtaDump = await this.getNlmtaFromIssn(doiInfo);
+      if (nlmtaDump) {
+        doiInfo.nlmta = nlmtaDump.nlmta;
+        doiInfo['issn-map'] = nlmtaDump.map;
```
Comment (Rytiggy) [inline app/components/workflow-basics.js]: This is getting shown to the user as `[object, object]` should be stringified or not displayed. 
screen shot 2018-06-25 at 12 07 43 pm

Comment (Rytiggy) [reply app/components/workflow-basics.js]: This will also need to be updated in the submission/detail route

Comment (Rytiggy) [reply app/components/workflow-basics.js]: After **those fixs** this PR is good to go

Comment (karenhanson) [reply app/components/workflow-basics.js]: my vote is don't show it! we already have ISSN showing

**Comment by jrmartino** [comment]
followed the above recipe, behavior was as expected

**Pull request merged**